src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -67,7 +70,7 @@ export class SignifyApi {
     "EBIFDhtSE0cM4nbTnaMqiV1vUIlcnbsqBMeVMmeGmXOu";
   static resolvedOobi: { [key: string]: any } = {};
 
-  private isOnline = false;
+  static isOnline = false;
```

Comment:
This shouldn't be static if `signifyClient` isn't static, I would revert this change

Reply:
I face some issues with the unit tests here so I changed it to static. The decorator in the unit test cannot read this value.

Reply:
hmm, I wouldn't change the implementation to satisfy the tests though, this might lead to bugs. What is the error in the test?

Reply:
When I called the `getKeriaOnlineStatus` in the signifyApi. It always returns `undefined`